packaging/os/pkgutil.py

```diff
@@ -133,7 +133,7 @@ def main():
         ),
         supports_check_mode=True
     )
-    name = module.params['name']
+    name = module.params['name'].split(',')
```

Comment:
Instead of splitting on `,` here, you should probably update the `argument_spec` so that `name` looks like:

```
name = dict(required = True, type='list')
```

`type="list"` will automatically accept either a list or a CSV, and will automatically explode into a list.